### ISSUE TITLE
[Request] add an option to automatically compress files into winrar

### ISSUE THREAD

**Issue by tophlock** [RipMeApp/ripme#1080]
I always use program like mangareader for pc and perfect viewer for android but both of them required the file to be winrar etc.

Currently I'm manually compress the file I download into winrar. 
I hope it is an easy task to implement

**Comment by rautamiekka** [comment]
It says on https://play.google.com/store/apps/details?id=com.rookiestudio.perfectviewer it supports CBZ/ZIP, CBR/RAR, 7Z/CB7, LZH, CBT/TAR.

And RAR is rather strictly a Window$ thing anyway as far as latest support is concerned.

**Comment by ealgase** [comment]
An option to automatically compress into 7z/tar would be nice (but not necessary).  They're both open formats.

**Comment by cyian-1756** [comment]
I did some work on this a few releases ago but haven't got around to documenting it yet

You can have ripme run a command when a ripme finished by setting `enable.finish.command` to true in the config and setting `finish.command` to the command you want to run.

There are 2 special values to take note of with finish.command %url% which will be replace by the url of the download album and %path% which will be replaced by the path to the downloaded files 

I still need to add a %ripdir% before it's easily usable for this however

**Comment by rautamiekka** [comment]
> An option to automatically compress into 7z/tar would be nice (but not necessary). They're both open formats.

Yes, both are open formats and not Window$-specific (if anything, 7z is partially Window$-specific (you need to install an older (or compile it yourself) p7-Zip) and TAR is NIX-specific), but TAR's "problem" is it's completely uncompressed while 7z can be both uncompressed and compressed (when you make the package at first, you can choose between zero compression or some compression all the way to ultimate compression, but when you add to the file you can mix things, which is sorta a bummer but also sorta cool).

The finishing command will be a good thing for some.
